Lesson_25_Functions/js/homework.js

```diff
@@ -0,0 +1,138 @@
+//1. Написать функцию, которая принимает 2 числа и возвращает -1, если первое меньше, чем второе; 1 – если первое больше, чем второе; и 0 – если числа равны.
+
+function getNumbers(num1, num2) {
+  let num1 = +prompt("Enter number num1:");
+  let num2 = +prompt("Enter number num2:");
+
+  if (num1 < num2) {
+    console.log(-1);
+  } else if (num1 > num2) {
+    console.log(1);
+  } else {
+    console.log(0);
+  }
```

Comment:
тут нужен `return`, чтоб функция возвращала эти значения
```suggestion
  if (num1 < num2) {
    return -1;
  } else if (num1 > num2) {
    return 1;
  } else {
    return 0;
  }
```